containers/test/run.sh

```diff
@@ -0,0 +1,35 @@
+CONTENT=$HOME
+ENTRYPOINT=""
+if [ "$1" != "" ]; then
+  if [ "$1" != "shell" ]; then
+    CONTENT=$1
+    shift
+  fi
+  if [ "$1" == "shell" ]; then
+    ENTRYPOINT="/bin/bash"
+  fi
+fi
+
+# disable EULA for the sake of tests
+mkdir -p $CONTENT/datalab/.config/eula
+
+# On linux docker runs directly on host machine, so bind to 127.0.0.1 only
```

Comment:
@ojarjur 
Just to clarify,  based on the comments above I understand that port binding to IP 127.0.0.1 is still needed. The obsolete part is that binding to 127.0.0.1 it is no longer limited to Linux only.  Can you confirm?